src/App.js

```diff
@@ -1,23 +1,337 @@
-import logo from './logo.svg';
-import './App.css';
+import './index.css';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { TailSpin } from 'react-loader-spinner';
```

Comment:
Totally stealing this and looking it up, I assume it's a loading animation. It is now mine. Thank.